deepchem/data/datasets.py

```diff
@@ -1751,15 +1949,13 @@ def transform(self, fn, **args):
   def select(self, indices, select_dir=None):
     """Creates a new dataset from a selection of indices from self.
 
-    TODO(rbharath): select_dir is here due to dc.splits always passing in
-    splits.
-
     Parameters
     ----------
     indices: list
       List of indices to select.
     select_dir: string
-      Ignored.
+      Used to provide same API as `DiskDataset`. Ignored since
+      `NumpYDataset` is purely in-memory.
```

Comment:
This is a method of ImageDataset, not NumpyDataset.

Reply:
Good catch, fixed.